src/main/java/com/stud/java/services/impl/CatGenerator.java

```diff
@@ -0,0 +1,48 @@
+package com.stud.java.services.impl;
+
+import com.stud.java.enums.Breed;
+import com.stud.java.enums.Color;
+import com.stud.java.repository.entity.CatEntity;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.UUID;
+
+public class CatGenerator {
+
+    public List<CatEntity> generate(int num) {
```

Comment:
Would you kindly, add CatGenerator interface in com.stud.java.services, that has 1 method
```List<CatEntity> generate(int num);```

Then, implement that interfacce with this class, and rename this class to CatGeneratorImpl